### PR TITLE
New package: imv-2.1.3.

### PR DESCRIPTION
**Pull request by inefarius**
Image viewer for X11/Wayland

### CODE REVIEW

File: srcpkgs/imv/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+build_style=gnu-makefile
+make_use_env="IMV_VERSION=${version}"
+makedepends="SDL2_ttf-devel freeimage-devel fontconfig-devel"
+short_desc="Image viewer for X11/Wayland"
+maintainer="Arvin Ignaci <arvin.ignaci@gmail.com>"
+license="GPL-2"
+homepage="https://github.com/eXeC64/imv"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=f0b9acd7cff1d2f7301cb5cffb261ace4d0af2003dc4a293d9a37e73074978b7
+nostrip_files="imv"
```
Comment (Vaelatern) [inline srcpkgs/imv/template]: Wrong.

Comment (Vaelatern) [reply srcpkgs/imv/template]: Very wrong.

File: srcpkgs/imv/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'imv'
+pkgname=imv
+version=2.1.3
+revision=1
+build_style=gnu-makefile
+make_use_env="IMV_VERSION=${version}"
```
Comment (Vaelatern) [inline srcpkgs/imv/template]: Highly wrong. Please read the documentation. Or the makefile scripts. There is a better way to do this.

**Comment by pullmoll** [comment]
@inefarius please take a look at the merged `srcpkgs/imv/template` for one way to handle a gnu-makefile build style which adds to `{C,LD}FLAGS`.

**Pull request closed without merging**